pyubee/__init__.py

```diff
@@ -168,6 +168,22 @@ def authenticate(self, url, username, password):
         ),
         'authenticator': Evw3226Authenticator
     },
+    'EVW321B': {
+        'url_session_active': '/HomePageMR4.asp',
+        'url_login': '/goform/loginMR4',
+        'url_logout': '/logout.asp',
+        # includes all devices, also WiFi
+        'url_connected_devices_lan': '/ConnectedDevicesMR4.asp',
+        # there is no separate page with WiFi devices
+        'url_connected_devices_wifi': None,
+        'regex_login': re.compile(r'name="loginUsername"'),
+        'regex_wifi_devices': None,
+        'regex_lan_devices': re.compile(
+            r'<td id="MACAddr">([0-9a-fA-F:]{17})</td>'  # mac address
```

Comment:
I would go with 

```python
            r'<td id="MACAddr">([0-9a-fA-F]{2}:[0-9a-fA-F]{2}:[0-9a-fA-F]{2}:[0-9a-fA-F]{2}:'  # mac address
            r'[0-9a-fA-F]{2}:[0-9a-fA-F]{2})</td>'  # mac address, cont'd
```

like we did for other models, it's more precise than: 

```python
           r'<td id="MACAddr">([0-9a-fA-F:]{17})</td>'  # mac address
```

Reply:
Done